From d16e81077808c9c898e35db0f4b8f60e0490bf09 Mon Sep 17 00:00:00 2001
From: Tobias Brunner <tobias@strongswan.org>
Date: Wed, 29 Jan 2020 11:05:30 +0100
Subject: [PATCH] pki: Remove unnecessary and problematic chunk_from_chars()
 usage in --signcrl

If the serial is not yet set, the same default value is set just below.

See 8ea13bbc5ccd for details on chunk_from_chars().

References #3249.

Origin: upstream, https://git.strongswan.org/?p=strongswan.git;a=commitdiff;h=d16e81077808c9c898e35db0f4b8f60e0490bf09
Reviewed-By: Lucas Kanashiro <kanashiro@ubuntu.com>
Last-Updated: 2020-05-22
---
 src/pki/commands/signcrl.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/pki/commands/signcrl.c b/src/pki/commands/signcrl.c
index dfe1ce0..60d880e 100644
--- a/src/pki/commands/signcrl.c
+++ b/src/pki/commands/signcrl.c
@@ -385,10 +385,6 @@ static int sign_crl()
 	}
 	else
 	{
-		if (!crl_serial.ptr)
-		{
-			crl_serial = chunk_from_chars(0x00);
-		}
 		lastenum = enumerator_create_empty();
 	}
 
-- 
2.7.4

